docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1911 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #58.            cand_idx_chunks = _get_chunks_ranges(non_trivial_cands)
</pre><p>I wonder if it makes any sense to move lines 58-65 into a separate function since both <code>_select_candidates</code>  and <code>_prune_candidates</code>  are doing the exact same thing. If they are not the same then please ignore me</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Right on! I will refactor it next time!